signup_page/style.css

```
@@ -0,0 +1,132 @@
+@import "/reset.css";
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    font-family: "Pretendard Variable", Pretendard, -apple-system, BlinkMacSystemFont, system-ui, Roboto, "Helvetica Neue", "Segoe UI", "Apple SD Gothic Neo", "Noto Sans KR", "Malgun Gothic", "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", sans-serif;
+}
+
+html, body {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    justify-content: center;
+}
```

💊 제안
font-family의 경우 전체 선택자로 모든 태그에 속성을 주기보다 html 태그에 적용하시고 자식들은 상속받도록 하는게 더 좋을 것 같아요.
margin 속성을 0으로 하는 속성은 reset.css와 겹치니 지우셔도 될 것 같습니다~

```suggestion
* {
    box-sizing: border-box;
}

html, body {
    width: 100%;
    height: 100%;
    display: flex;
    justify-content: center;
    font-family: "Pretendard Variable", Pretendard, -apple-system, BlinkMacSystemFont, system-ui, Roboto, "Helvetica Neue", "Segoe UI", "Apple SD Gothic Neo", "Noto Sans KR", "Malgun Gothic", "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", sans-serif;
}
```